tests/ImageSharp.Tests/Formats/WebP/ImageExtensionsTest.cs

```diff
@@ -13,12 +13,15 @@ namespace SixLabors.ImageSharp.Tests.Formats.Webp
     [Trait("Format", "Webp")]
     public class ImageExtensionsTest
     {
+        private readonly Configuration configuration;
+
         public ImageExtensionsTest()
         {
-            Configuration.Default.ImageFormatsManager.AddImageFormat(WebpFormat.Instance);
-            Configuration.Default.ImageFormatsManager.AddImageFormatDetector(new WebpImageFormatDetector());
-            Configuration.Default.ImageFormatsManager.SetDecoder(WebpFormat.Instance, new WebpDecoder());
-            Configuration.Default.ImageFormatsManager.SetEncoder(WebpFormat.Instance, new WebpEncoder());
+            this.configuration = new Configuration();
+            this.configuration.ImageFormatsManager.AddImageFormat(WebpFormat.Instance);
```

Comment:
Does anyone else find the existing registration API here clunky?

Reply:
We can define helpers like `configuration.AddWebp()`.